src/main/scala/scalikejdbc/athena/Config.scala

```diff
@@ -64,21 +69,44 @@ class Config(dbName: Any) {
 object Config {
   val Url = "url"
   val Driver = "driver"
-
-  val S3StagingDir = "s3_staging_dir"
-  val S3StagingDirPrefix = "s3_staging_dir_prefix"
-
-  val QueryResultsEncryptionOption = "query_results_encryption_option"
-  val QueryResultsAWSKmsKey = "query_results_aws_kms_key"
-  val AWSCredentialsProviderClass = "aws_credentials_provider_class"
-  val AWSCredentialsProviderArguments = "aws_credentials_provider_arguments"
-  val MaxErrorRetries = "max_error_retries"
-  val ConnectionTimeout = "connection_timeout"
-  val SocketTimeout = "socket_timeout"
-  val RetryBaseDelay = "retry_base_delay"
-  val RetryMaxBackoffTime = "retry_max_backoff_time"
-  val LogPath = "log_path"
-  val LogLevel = "log_level"
+  val ReadOnly = "readOnly"
+
+  val S3OutputLocation = "S3OutputLocation"
+  val S3OutputLocationPrefix = "S3OutputLocationPrefix"
+
+  val AwsCredentialsProviderArguments = "AwsCredentialsProviderArguments"
+  val AwsCredentialsProviderClass = "AwsCredentialsProviderClass"
+  val BinaryColumnLength = "BinaryColumnLength"
+  val ComplexTypeColumnLength = "ComplexTypeColumnLength"
+  val ConnectionTest = "ConnectionTest"
+  val ConnectionTimeout = "ConnectTimeout"
```

Comment:
I think that it is better to match val name to config option name.
```scala
val ConnectTimeout = "ConnectTimeout"
```

Reply:
Uh-oh. sorry. to correct